_posts/2019-11-27-cockpit-208.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 208
+author: pitti
+date: 2019-11-27
+tags: cockpit linux
+slug: cockpit-208
+category: release
+summary: Cockpit with Machines, file system creation, and crash reporting improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.  Here are the release notes from version 208.
+
+### Storage: Drop "default mount point" concept
```

Comment:
I'm not sure about this change. But what if  we focus on unifying fstab and mtab?

```suggestion
### Storage: Unify saved and current mounts
```

Reply:
The "drop default mount point" was the main change, and the static/runtime disparity more of a by-product. The main result from usability review was that few people understand this magic "default mount point" concept.